src/components/component-renderer/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Paragraph from './paragraph';
+import Link from './link';
+import Strong from './strong';
+import Title1 from './title1';
+import Title2 from './title2';
+import Title3 from './title3';
+import Content from '../content';
```

Comment:
Do we need the content component? Once we move to declaring each component that the content renderer can handle I think this can be removed.